packages/flutter/test/material/bottom_navigation_bar_test.dart

```diff
@@ -1150,6 +1151,35 @@ void main() {
     expect(tester.getSize(find.text(label).last), equals(const Size(168.0, 56.0)));
   });
 
+  testWidgets('BottomNavigationBar shows custom tool tips if toolTip is provided in BottomNavigationBarItem', (WidgetTester tester) async {
```

Comment:
Thanks for providing a test! Could you add another that verifies the tooltip doesn't show up when you set the new parameter to null?